tool/tsh/tsh.go

```diff
@@ -689,6 +692,46 @@ func onListNodes(cf *CLIConf) {
 	}
 }
 
+func showNodes(nodes []services.Server, verbose bool) {
+	switch verbose {
+	// In verbose mode, print everything on a single line and include the Node
+	// ID (UUID). Useful for machines that need to parse the output of "tsh ls".
+	case true:
+		t := asciitable.MakeTable([]string{"Node Name", "Node ID", "Address", "Labels"})
```

Comment:
I'm going back and fourth on camel case / capital case. Since we recommend customers to use node-id. I think we should reflect this in the table.

Reply:
I believe the current usage is consistent with how things are done elsewhere.  For example, `tctl users add` accepts the `--roles` flag, but `tctl users ls` has a column named `Roles`.

Reply:
👍 sounds good, it's more a nit than anything else.